XboxHelper.py

```diff
@@ -246,14 +250,15 @@ def resume_fifo_pusher(self):
         if self.delay():
             pass
 
-    def allow_populate_fifo_cache(self):
+    def allow_populate_fifo_cache(self, wait_time=0.05):
```

Comment:
> I'm assuming you're talking about the `wait_time` specifically?

Yes.

> As far as I can tell the current xbdm transport layer is limited by having a timeout on hack invocations

I don't remember. But nv2a-trace clearly needs faster networking and more code running on the Xbox.
Usually I'd not care if capturing a scene took an hour or two as long as it captured all necessary detail.

I don't think I've ever had problems with the games time diverging during a capture, but I do remember crashes when returning from the capture, because the games internal timers diverged too much.

Reply:
(Sorry, I'm responding to comments in reverse order so will be somewhat brief here to reduce duplication)

The issue I've run into (and other recent users) is a bit the general usability issue of a capture taking very long (I've definitely gone well over two hours for a frame) but more so transient instability that makes it feel like the longer the capture takes, the more likely it is to crash somewhere along the way, requiring an additional attempt.

I don't think it's necessarily a huge deal if everything crashes once the capture is complete (given that there are #fixme's regarding returning to original state I'd worry that a second capture in a session might not be entirely trustable), but it is extremely frustrating if 90 minutes into an attempt there's a failure in xboxpy retrieving memory due to a networking timeout. Most of my motivation in attempting to reduce waits is around the hope that less time spent capturing means less opportunity for the transient failure to manifest coupled with the assumption that having to restart due to a failure 30 minutes in feels less discouraging than the same failure 60 minutes in :)